Makefile.toml

```diff
@@ -0,0 +1,98 @@
+[tasks.frontend]
+command = "./client/build"
+toolchain = "nightly"
```

Comment:
I need to investigate whether we can set this globally somehow (and require a different toolchain per task **if** needed)

Reply:
> cargo-make supports setting the toolchain to be used when invoking commands and installing rust dependencies by setting the toolchain attribute as part of the task definition.

To me this seems as it's not possible to add a global option here. However having the toolchain explicitly defined is still better than getting weird compiler issues :)